utils/apiCalls/authorization.ts

```diff
@@ -0,0 +1,41 @@
+import axios, { AxiosRequestConfig } from 'axios';
+import { FIREBASE_API_KEY } from 'react-native-dotenv';
```

Comment:
One more thing that I forgot. Please document this env variable in the .env.example file.

Reply:
OK, I see it's resolved :ok_hand: